### ISSUE TITLE
Add a browser to the minimal ISO

### ISSUE THREAD

**Issue by Noxbru** [Antergos/antergos-iso#86]
Hi,

When users have problems with the installation we usually ask them to share the contents of the log file using pastebin or similar. But this is very difficult in the minimal ISO because it doesn't have a broswer (note, this also makes it difficult to check documentation when installing).

So, I propose to add a minimal browser to the minimal ISO. I have been checking different posibilities and 'epiphany' only adds about 11MiB and I have been able to put some text in pastie.se and check the resulting URL easily.

Thanks in advance!
Noxbru

**Comment by karasu** [comment]
The testing version has midori on it... but we still have to merge both versions (master and testing).

I'll leave this open as a reminder.

**Comment by Noxbru** [comment]
Midori was my second posibility, as it is a bit more big (~20MiB more) But it is nice to know that this is a work in progress.

Can we expect that this testing branch will be merged for the ISO for next month?

**Comment by Yoshi2889** [comment]
Doesn't epiphany pull in a large runtime of libraries?

**Comment by Noxbru** [comment]
@Yoshi2889 As said in the original message, it only adds about 11MiB (the sum of all the packages to be installed as reported by pacman), tested in the 17.5 ISO. It may have changed, though, so I will retest again with the latest minimal-iso and report again.

**Comment by Yoshi2889** [comment]
I thought you meant the package itself only. Is the webkit2gtk runtime or whatever they use now already in the ISO?

**Comment by Noxbru** [comment]
@Yoshi2889 Yes, I reported the full installation size needed. And yes, `webkit2gtk` is already in the iso because it is a dependency of `cnchi`.
